src/core/enums/ActionGroup.enum.h

```diff
@@ -56,7 +56,7 @@ enum ActionGroup {
 
     GROUP_HIGHLIGHT_POSITION,
 
-    GROUP_SETSQUARE,
+    GROUP_GEOMETRY_TOOL,
```

Comment:
Question: do we want to only ever have at most one geometric tool up?

Reply:
Yes, at most one geometry tool. I see no benefit in having several geometry tools up at the same time. It's only distracting. On real paper you sometimes need two tools (ruler and set square) for parallel translation, but digitally that's the easiest thing to do (via one finger touch or arrow keys).